### PR TITLE
[NOT AZURE] Azure Stack Gallery Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

This is **NOT** for Azure.

https://github.com/Azure/azure-rest-api-specs/pull/2780

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/GalleryAdmin/Gallery.Admin/Microsoft.AzureStack.Management.Gallery.Admin.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <PackageId>Microsoft.AzureStack.Management.Gallery.Admin</PackageId>
+    <Description>Provides developers with libraries for the gallery admin under AzureStack to manage gallery items.</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.AzureStack.Management.Gallery.Admin</AssemblyName>
+    <PackageTags>Microsoft Azure Stack;Gallery;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the AzureStack Gallery Admin SDK.  Included in this release is the ability to list, register, get, and delete gallery items.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
```
Comment (shahabhijeet) [inline src/AzureStack/GalleryAdmin/Gallery.Admin/Microsoft.AzureStack.Management.Gallery.Admin.csproj]: @deathly809 missing footer includes

File: src/AzureStack/GalleryAdmin/Gallery.Admin/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Microsoft Azure .NET SDK")]
+[assembly: AssemblyCopyright("Copyright © Microsoft")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+// Version information for an assembly consists of the following four values:
```
Comment (shahabhijeet) [inline src/AzureStack/GalleryAdmin/Gallery.Admin/Properties/AssemblyInfo.cs]: @deathly809 remove commented code

File: src/AzureStack/GalleryAdmin/Gallery.Admin.Tests/Gallery.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Gallery.Admin.Tests</PackageId>
+    <Description>Gallery.Admin.Tests Class Library</Description>
+    <AssemblyName>Gallery.Admin.Tests</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1.2</TargetFrameworks>
```
Comment (shahabhijeet) [inline src/AzureStack/GalleryAdmin/Gallery.Admin.Tests/Gallery.Tests.csproj]: @deathly809 any reason you need these additional frameworks

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/AzureStack/GalleryAdmin/Gallery.Admin/Microsoft.AzureStack.Management.Gallery.Admin.csproj (hunk 1 above)
Comment (deathly809) [reply src/AzureStack/GalleryAdmin/Gallery.Admin/Microsoft.AzureStack.Management.Gallery.Admin.csproj]: Fixed.

File: src/AzureStack/GalleryAdmin/Gallery.Admin.Tests/Gallery.Tests.csproj (hunk 3 above)
Comment (deathly809) [reply src/AzureStack/GalleryAdmin/Gallery.Admin.Tests/Gallery.Tests.csproj]: Fixed.

File: src/AzureStack/GalleryAdmin/Gallery.Admin/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (deathly809) [reply src/AzureStack/GalleryAdmin/Gallery.Admin/Properties/AssemblyInfo.cs]: Fixed.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**